sr_demos/scripts/sr_demos/demo.py

```diff
@@ -316,10 +325,28 @@ def check_touched_finger(self):
             touched_finger = self.tactiles[0].confirm_touched()
         return touched_finger
 
+    def _enable_touch_detection(self, detect: bool):
```

Comment:
I think you are missing the output hint
```suggestion
    def _enable_touch_detection(self, detect: bool) -> None:
```